fuse/lafayette_pd.py

```diff
@@ -43,24 +43,33 @@ def fuse_events(cprr, pprr):
 
 
 if __name__ == '__main__':
-    cprr = pd.read_csv(data_file_path(
+    cprr_20 = pd.read_csv(data_file_path(
         'match/cprr_lafayette_pd_2015_2020.csv'
     ))
+    cprr_14 = pd.read_csv(data_file_path(
+        'match/cprr_lafayette_pd_2009_2014.csv'
+    ))
     pprr = pd.read_csv(data_file_path(
         'clean/pprr_lafayette_pd_2010_2021.csv'
     ))
     post_events = pd.read_csv(data_file_path(
         'match/post_event_lafayette_pd_2020.csv'
     ))
-    events_df = fuse_events(cprr, pprr)
+    events_df = fuse_events(cprr_20, cprr_14, pprr)
     events_df = rearrange_event_columns(pd.concat([
         post_events,
         events_df
     ]))
     per = fuse_personnel(
         pprr,
-        cprr[['uid', 'first_name', 'last_name']],
-        cprr[['investigator_uid', 'investigator_first_name', 'investigator_last_name']].rename(columns={
+        cprr_20[['uid', 'first_name', 'last_name']],
+        cprr_20[['investigator_uid', 'investigator_first_name', 'investigator_last_name']].rename(columns={
+            'investigator_uid': 'uid',
+            'investigator_first_name': 'first_name',
+            'investigator_last_name': 'last_name',
+        }),
+        cprr_14[['uid', 'first_name', 'last_name']],
+        cprr_14[['investigator_uid', 'investigator_first_name', 'investigator_last_name']].rename(columns={
```

Comment:
What with the lafayette changes?

Reply:
They are a product of the merge. I notice errors. I'll fix them in the bugs/misc branch.